django/contrib/staticfiles/templatetags/staticfiles.py

```diff
@@ -1,16 +1,29 @@
+import warnings
+
 from django import template
-from django.contrib.staticfiles.storage import staticfiles_storage
 from django.templatetags.static import StaticNode
+from django.templatetags.static import static as _static
+from django.utils.deprecation import RemovedInDjango20Warning
 
 register = template.Library()
 
 
 def static(path):
-    return staticfiles_storage.url(path)
+    # Implementation has been moved to django.templatetags.static.
```

Comment:
@codingjoe I think we probably want to leave a note in a comment (FIXME?) to deal with this later. It's a bit ugly, but introducing more deprecation warnings infrastructure feels overkill. We can retain the documentation that this is now deprecated, and introduce it as a warning later, at 1.11 or 2.0. @timgraham is that what you were thinking?